src/opm/output/eclipse/WriteInit.cpp

```diff
@@ -32,7 +32,6 @@
 #include <opm/output/eclipse/WriteRestartHelpers.hpp>
 
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
-#include <opm/parser/eclipse/EclipseState/EndpointScaling.hpp>
```

Comment:
Is there a reason for removing this header?  We definitely use the [`EndpointScaling`](https://github.com/OPM/opm-common/blob/a1ee2f3b07b6b78d44d1a1b6b46257cebef7c8cb/opm/parser/eclipse/EclipseState/EndpointScaling.hpp#L28) class when collecting and emitting the [end-point scaling flags](https://github.com/OPM/opm-common/blob/a1ee2f3b07b6b78d44d1a1b6b46257cebef7c8cb/src/opm/output/eclipse/WriteInit.cpp#L261-L266) of the [`LOGIHEAD`](https://github.com/OPM/opm-common/blob/a1ee2f3b07b6b78d44d1a1b6b46257cebef7c8cb/src/opm/output/eclipse/WriteInit.cpp#L280) vector.

Reply:
You are right.  It is the IDE I am using show this header file is not used, and I went through quickly, did not find it used, which was a mistake.

FYI, and I just checked the standard from the IDE, it is based on the following standard to mark this header file not used. 
```
Detect not directly used
an include directive is unused if declarations from it are not used in the file directly. This works best with the "Include What You Use" policy.
```